compose/ui/ui/src/desktopTest/kotlin/androidx/compose/ui/window/TestUtils.kt

```diff
@@ -164,3 +170,127 @@ internal class WindowTestScope(
         exceptionHandler.throwIfCaught()
     }
 }
+
+internal class ComponentTestMethods<C: Component, S: Any>(
+    val create: () -> C,
+    val setContent: C.(content: @Composable S.() -> Unit) -> Unit,
+    val display: C.() -> Unit,
+    val dispose: C.() -> Unit
+)
+
+internal fun <C: Component, S: Any> componentOrientationModifiesLayoutDirection(
```

Comment:
Better to not place specific tests in TestUtils. We can move them to something like ComponentOrientationTests

Reply:
ok

Reply:
It seems merged without moving it out of TestUtils

Reply:
Oops. Forgot to push before merging. I'll push the difference as a separate commit.

Reply:
See https://github.com/JetBrains/compose-multiplatform-core/pull/790